### ISSUE TITLE
No posts found?

### ISSUE THREAD

**Issue by codeinteger6** [irfancharania/fb-feed-gen#8]
I'm trying to get an RSS feed of the [Animax Asia Facebook page](https://facebook.com/AnimaxAsiaTV). But for some reason it's saying no posts.

![image](https://user-images.githubusercontent.com/44692189/59150096-ed47f700-8a3f-11e9-8235-eb704d0dec15.png)

**Comment by irfancharania** [comment]
The example urls still work just fine. For some reason, the lxml parser is unable to parse the Animax Asia page. I'm not sure why... there may be some character there that is throwing it off? I suspect once the offending post is out of the main page, then it will start working again...

**Comment by codeinteger6** [comment]
@irfancharania, is this issue fixed now?

**Comment by irfancharania** [comment]
I haven't come across it again... so closing it

**Comment by eniac111** [comment]
Hey,

I'm experiencing it with every single page. @irfancharania do you still use the script? 

Cheers!

**Comment by irfancharania** [comment]
I'm still using it, and it works for both the examples on the main page.
This app only works for publicly available pages. Are you sure the pages you're supplying are posting publicly?

**Comment by eniac111** [comment]
Yes, I am sure, strange. Anyway, I have found a similar php script which is working. The parsing part is boring :)

Regards

**Comment by m040601** [comment]
Hi same problem here

> ... and it works for both the examples on the main page.

For me it doesnt, same error

>  ...Are you sure the pages you're supplying are posting publicly?

Yes, 100% sure. Tried with many different pages

**Comment by irfancharania** [comment]
What url are you trying?

To deploy the application, you would click the heroku button, and go through the wizard and at the end it tells you the app url. You proceed to the url, put in the user/page name and it should generate the atom feed.

When I just tried, it failed initially because of outdated dependencies with security issues. I updated the dependencies and tried again, it deploys and works for me for the example sites...

You've got the correct url in your log... do you see any posts when you go to FB page directly?
https://mbasic.facebook.com/IslandFarmFresh/

I haven't added any logging to the app...

**Comment by Dylan700Manes** [comment]
Hello, the generator does not seem to work with the following users:
IslandFarmFresh
elysee.fr
While,
[https://mbasic.facebook.com/elysee.fr](https://mbasic.facebook.com/elysee.fr) 
[https://mbasic.facebook.com/IslandFarmFresh](https://mbasic.facebook.com/IslandFarmFresh) 
Returns a result

**Comment by irfancharania** [comment]
I just created a new Heroku app using the deploy button on the main page, and checked both IslandFarmFresh and elysee.fr. Both come up with posts for me...

**Comment by Cheng-CM** [comment]
Hello, seems like the problem is on Facebook's side, I tried to return the FB page back to the app and it's just a login page. 
Tried switching user agents, not working so far.

**Comment by juanjosepablos** [comment]
It does not work for me either. I think that the problem is that you need to by pass the cookies requirement.
